package.json

```diff
@@ -15,7 +15,7 @@
     "elasticsearch": "^9.0.2",
```

Comment:
Bump version on line 3. Recommend `2.0.0` (or `3.0.0`, really, if we merge #129 first) as this is a backwards-incompatible change. See https://semver.org/#summary for more info.

Reply:
Decided to keep at 2.0.0 from the es update. See #129 for discussion.